tools/esdlcmd/esdlcmd_manifest.cpp

```diff
@@ -0,0 +1,1409 @@
+                                if (isManifestNodeNamed(stag, MANIFEST_TAG_SVC_BINDING))
+                                    type = processServiceBinding(xpp.get(), stag);
+                                else if (isManifestNodeNamed(stag, MANIFEST_TAG_SVC_DEFINITION))
+                                    type = processServiceDefinition(xpp.get(), stag);
+                                else
+                                {
+                                    appendStartTag(xpp.get(), stag);
+                                    level++;
+                                }
+                                break;
+                            }
+
+                            case XmlPullParser::CONTENT:
+                                appendNonWhitespaceContent(xpp.get());
+                                break;
+
+                            case XmlPullParser::END_TAG:
+                                xpp->readEndTag(etag);
+                                if(!isRootEndTagNamed(etag, MANIFEST_TAG_MANIFEST, level))
+                                    appendEndTag(etag);
+                                level--;
+                                break;
+
+                            case XmlPullParser::END_DOCUMENT:
+                                level = 0;
+                                break;
+                        }
+
+                        lastType = type;
+
+                        #ifdef _MANIFEST_DEBUG_
+                        UERRLOG("processBinding while:(...%s)", binding.length()<100 ? binding.str() : binding.str()+binding.length()-100);
+                        #endif
+                    }
+                    while (level > 0);
+
+                    if (isBundle)
+                        appendEndTag(MANIFEST_TAG_BUNDLE);
+                    if (isBundle && !foundServiceDefn)
+                        UWARNLOG("Generating EsdlBundle output without ESDL Service Definition. This may cause runtime issues.");
+
+                    #ifdef _MANIFEST_DEBUG_
+                    UERRLOG("processBinding final:(%s)", binding.str());
+                    #endif
+                }
+                else
+                {
+                    UERRLOG("Error reading manifest file (%s): empty or does not exist.", optManifestPath.str());
+                    result = false;
+                }
+            }
+            catch (IException* e)
+            {
+                StringBuffer msg;
+                e->errorMessage(msg);
+                UERRLOG("Error reading manifest file (%s): %s", optManifestPath.str(), msg.str());
+                result = false;
+                e->Release();
+            }
+            catch (XmlPullParserException& xe)
+            {
+                const char* msg = xe.what();
+                UERRLOG("Error reading manifest file (%s) at line #%d, column #%d: %s", optManifestPath.str(), xe.getLineNumber(), xe.getColumnNumber(), msg ? msg : "Unknown");
```

Comment:
Could be a misleading message since multiple parsers can be used with file inclusion.

Reply:
Removed use of additional parsers.